src/utils/subscriberBreaches.ts

```diff
@@ -74,7 +74,13 @@ export async function getSubBreaches(
     const breachResolution = subscriber.breach_resolution
```

Comment:
Yea I think this is off.. it should just be something like:
`subscriber.breach_resolution?.[email.email] ?? {}`

Reply:
Good catch!

Reply:
Cool, added in 143426815. Catching stuff like this is what the coverage report is useful for!